sdk/src/client/secret/ledger_nano.rs

```diff
@@ -85,11 +88,7 @@ pub enum Error {
     ExpirationDeadzone,
 }
 
-impl From<crate::types::block::Error> for Error {
-    fn from(error: crate::types::block::Error) -> Self {
-        Self::Block(Box::new(error))
-    }
-}
+crate::impl_from_error_via!(Error via BlockError: OutputError, UnlockError, SignatureError);
```

Comment:
Why is it that sometimes we use `derive_more::From` + `#[from]` and sometimes this macro?

Reply:
Ah don't mind me, got it